index.js

```diff
@@ -840,6 +845,16 @@ function getValue(object) {
 }
 
 /*
+ * Strip string from leading/trailing space character, including space,
+ * tab, form feed, line feed, etc.
+ * @param string {string} - String to strip.
+ * @returns {string} - striped string
+*/
+function strip(string) {
+  return string.replace(/^\s*|\s*$/g, '');
```

Comment:
Fixed
Good catch :+1: 
For some reason I thought `trim` strip only whitespaces and not tabs and newlines.
But actually it strip exactly same things that `\s` do.